src/dune_config/config.ml

```diff
@@ -168,3 +168,24 @@ let threaded_console =
   register t;
   t
 ;;
+
+let threaded_console_refresh_rate =
+  let t =
+    { name = "threaded_console_refresh_rate"
+    ; of_string =
+        (fun x ->
+          match Int.of_string x with
+          | Some x ->
+            (match x > 0 && x <= 1000 with
+             | true -> Ok x
+             | false -> Error (sprintf "value must be between 1 and 1000"))
+          | None -> Error (sprintf "could not parse %S as an integer" x))
+    ; value =
+        (match Env.get Env.initial "INSIDE_EMACS" with
```

Comment:
It's not good to have to copy paste this variable

Reply:
`INSIDE_EMACS` lives inside `dune_util` and I didn't want to have to depend on it.

Reply:
You can make `Dune_threaded_console.make` take this parameter and then thread it pass it from whoever is initializing the console.

Reply:
Done.